dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -161,6 +163,8 @@ void Trace::addAppenders_(const std::vector<TraceAppender>& appenders) {
     boost::shared_ptr< FileSink > sink(new FileSink(
       keywords::file_name = appenders[i].getFilePath(),
       keywords::open_mode = mode));
+    SinkConfiguration sinkConfiguration = {appenders[i], sink, currentId};
+    tagToSinkConfigurationMap_.insert(std::make_pair(appenders[i].getTag(), sinkConfiguration));
```

Comment:
I don't understand why this is needed?

Reply:
It's to get the sink with the VARIABLES tag and remove it when we want to overwrite the VARIABLES log file.